.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [ 1, {
+      "js": "always", "json": "always"
+    }]
+  },
+  "ignorePatterns": [
+    "dist/",
+    "build/"
+  ]
+}
```

Comment:
- [ ] Please make sure to work more on the responsiveness of the website and try to align the item's center I think this would be better
![image](https://user-images.githubusercontent.com/74744787/181839319-5c453d32-201d-405e-8aab-843ae15bd340.png)

Reply:
Fixed